modules/ROOT/pages/hdfs/hdfs-connector-reference.adoc

```diff
@@ -1,649 +1,603 @@
+| Name Node Uri a| String |  The name of the file system to connect to. It is passed to HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overriden by values in *Configuration Resources* and *Configuration Entries*. |  | x
+| Configuration Resources a| Array of String |  A java.util.List of configuration resource files to be loaded by the HDFS client. Here you can provide additional configuration files, for example, core-site.xml. |  | 
+| Configuration Entries a| Object |  A java.util.Map of configuration entries to use by the HDFS client. Here you can provide additional configuration entries as key and value pairs. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
 |===
+
 [[hdfs_simple]]
 ===== Simple
 
 
-
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String |  User identity that Hadoop uses for permissions in HDFS.  When Simple Authentication is used, Hadoop requires the user to be set as a System Property called HADOOP_USER_NAME. If you fill this field then the connector will set it for you, however you can set it by yourself. If the variable is not set, Hadoop will use the current logged in OS user. |  |
-| Name Node Uri a| String |  The name of the file system to connect to. It is passed to HDFS client as the {FileSystem#FS_DEFAULT_NAME_KEY} configuration entry. It can be overriden by values in configurationResources and configurationEntries. |  | x
-| Configuration Resources a| Array of String |  A java.util.List of configuration resource files to be loaded by the HDFS client. Here you can provide additional configuration files. (for example, core-site.xml) |  |
-| Configuration Entries a| Object |  A java.util.Map of configuration entries to be used by the HDFS client. Here you can provide additional configuration entries as key/value pairs. |  |
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
+| Username a| String |  User identity that Hadoop uses for permissions in HDFS. When Simple Authentication is used, Hadoop requires the user to be set as a System Property called HADOOP_USER_NAME. If you fill this field then the connector will set it for you, however you can set it by yourself. If the variable is not set, Hadoop will use the current logged in OS user. |  | 
```

Comment:
This is confusing - what is the connector setting for you if you fill in the field?

Remove future tense.